spec/models/book_spec.rb

```diff
@@ -1,5 +1,101 @@
 require 'rails_helper'
+require 'faker'
+require 'rspec-rails'
 
-RSpec.describe Book, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Book, type: :model do
+  it do
```

Comment:
There's no need for the blocks, since they take only one line you can do:
`it should validate_presence_of(:genre)`